Assets/Scripts/Tank/TankController.cs

```diff
@@ -42,16 +46,19 @@ public void ApplyDamage(float damage)
                 TankModel.Health = TankModel.Health-damage;
             }
         }
+        // Sets model to view stats
         public void SetModelToView(TankView tank)
         {
             tank.Speed = TankModel.Speed;
             tank.Health = TankModel.Health;
         }
-        //public void DeathEvent()
-        //{
-        //    Debug.Log("Enemy player Dead");
-        //}
-
+        public void ResetController(TankModel tankmodel)
+        {
+            TankView.gameObject.SetActive(true);

```

Comment:
create a Reset function in the view and call that from the controller 
what happens inside the view when reset is called (setting GO to true) is internal to the view and controller should not know about it